tests/PKCE.test.ts

```diff
@@ -206,3 +207,17 @@ describe('Test storage types', () => {
     expect(localStorage.getItem('pkce_code_verifier')).not.toEqual(null);
   });
 });
+
+describe('Test additional cors options', () => {
+  it('Should have set the cors options correctly', async () => {
+    const instance = new PKCE(config);
+    const opts = instance.enableCorsCredentials(true)
+    expect(opts.credentials).toEqual('include')
+    expect(opts.mode).toEqual('cors')
```

Comment:
These tests are not testing that the code is working when making requests. Please update to run tests against mocked fetch calls.

Reply:
Pushed a change to enable this. I placed it in the existing group and extended the mockRequest in that group to take an additional optional parameter. This was just to reduce the amount of mocked code.